apps/fluent-tester/src/E2E/README.md

```diff
@@ -24,7 +30,18 @@
    - C:\repo\fluentui-react-native>`cd apps\win32`
    - C:\repo\fluentui-react-native\apps\win32> `yarn bundle`
 3. Run E2E tests
-   - C:\repo\fluentui-react-native\apps\win32> `yarn run e2etest`
+   - C:\repo\fluentui-react-native\apps\win32> `yarn e2etest`
+
+## UWP Steps
+
+1. Follow the same step as #1 above.
```

Comment:
Something like `1. Follow step #1 from the win32 steps, replacing win32 with windows" might be a little better :)

Reply:
ignore the win32 with windows part

Reply:
Changing wording in newest commit